.github/workflows/pr_cloud.yaml

```diff
@@ -1,16 +1,15 @@
-name: pr_test
+name: pr_cloud
 
 on:
   pull_request:
-    branches: main
+    branches:
+      - 'main'
+    paths:
+      - 'cloud/**'
```

Comment:
These should run when tests change (or at least, when pr_cloud.yaml changes)

Reply:
done